.github/workflows/check-common-flow-sync.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Deps checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
What is this flow?

Reply:
Because GHA does not allow referring to `container:` without supplying a container, I had to duplicate this file, with specific differences at the beginning of the file.
As an alternative (until GHA will support not referring to a container and behaving as running directly on the runner), I made check-common-flow-sync.yml flow, which verifies that the logic part of both files is in sync, making it impossible to edit one and not the other, without failing the CI.